modulefiles/core/esma_cmake/esma_cmake.lua

```diff
@@ -0,0 +1,21 @@
+help([[
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+conflict(pkgName)
+
+local opt = os.getenv("HPC_OPT") or os.getenv("OPT") or "/opt/modules"
+
+local base = pathJoin(opt,"core",pkgName,pkgVersion)
+
+prepend_path("CMAKE_MODULE_PATH", base, ";")
```

Comment:
See the comment in the CMakeModules modulefile.

Reply:
Done in f9f91faba8371301c2248bf53303fd6783579cba.